src/base/static/components/form-fields/messages/index.js

```diff
@@ -0,0 +1,41 @@
+import messages from "../../../language-module";
+
+const EN_US = require("./en_US.json");
+
+messages.addResourceBundle(
```

Comment:
This seems odd to me how this line is repeated for each component's bundle. Is that necessary? Is it an intentional pattern?

Reply:
Turns out this isn't necessary. We can import nested structures and i18next will automatically sub-namespace them. 👍